documentation/source/quickstart.rst

```diff
@@ -4,291 +4,188 @@
-a :ref:`supported simulator<simulator-support>`) and cocotb itself (``pip install cocotb``) available.
-
-Download and extract the cocotb source files according to the *release version* you are using from
-https://github.com/cocotb/cocotb/releases - you can check your cocotb version with ``cocotb-config --version``.
-
-The sources for cocotb's *development version* are available from https://github.com/cocotb/cocotb.
-See `Installing the Development Version <https://docs.cocotb.org/en/latest/install_devel.html>`_ for more details.
-
-The following lines are all you need to run a first simulation with cocotb:
-
-.. code-block:: bash
-
-    cd cocotb/examples/simple_dff
-    make
-
-This was running with the default simulator, Icarus Verilog,
-but selecting a different simulator is as easy as:
-
-.. code-block:: bash
-
-    make SIM=vcs
-
-
-Running the same example as VHDL
---------------------------------
-
-The ``simple_dff`` example includes both a VHDL and a Verilog :term:`RTL` implementation.
-The cocotb testbench can execute against either implementation using :term:`VPI` for
-Verilog and :term:`VHPI`/:term:`FLI` for VHDL.  To run the test suite against the VHDL
-implementation, use the following command (a :term:`VHPI` or :term:`FLI` capable simulator must
-be used):
+a :ref:`supported simulator<simulator-support>`) and cocotb itself (``pip install cocotb``) now.
 
-.. code-block:: bash
-
-    make SIM=ghdl TOPLEVEL_LANG=vhdl
-
-
-Using cocotb
-============
-
-A typical cocotb testbench requires no additional :term:`HDL` code (though nothing prevents you from adding testbench helper code).
-The Design Under Test (:term:`DUT`) is instantiated as the toplevel in the simulator
-without any wrapper code.
-Cocotb drives stimulus onto the inputs to the :term:`DUT` and monitors the outputs
-directly from Python.
-
-
-Creating a Makefile
--------------------
-
-To create a cocotb test we typically have to create a Makefile.  Cocotb provides
-rules which make it easy to get started.  We simply inform cocotb of the
-source files we need compiling, the toplevel entity to instantiate and the
-Python test script to load.
-
-.. code-block:: makefile
-
-    VERILOG_SOURCES = $(PWD)/submodule.sv $(PWD)/my_design.sv
-    # TOPLEVEL is the name of the toplevel module in your Verilog or VHDL file:
-    TOPLEVEL=my_design
-    # MODULE is the name of the Python test file:
-    MODULE=test_my_design
-
-    include $(shell cocotb-config --makefiles)/Makefile.sim
+.. _quickstart_creating_a_test:
 
-We would then create a file called ``test_my_design.py`` containing our tests.
+Creating a Test
+===============
 
+A typical cocotb testbench requires no additional :term:`HDL` code.
+The :term:`DUT` is instantiated as the toplevel in the simulator
+without any HDL wrapper code.
 
-.. _quickstart_creating_a_test:
+The test is written in Python.
 
-Creating a test
----------------
+Cocotb wraps your HDL top level with the handle you pass it,
```

Comment:
Nit: you have "top level" here, and "toplevel" in the previous paragraph. Choose one.

Reply:
Used `toplevel` since our variable is called `TOPLEVEL` and not `TOP_LEVEL`.